skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/BrevmalService.kt

```diff
@@ -46,7 +46,7 @@ class BrevmalService(
     }
 
     private suspend fun hentMaler(sakstype: Sakstype, includeEblanketter: Boolean): Sequence<LetterMetadata> = coroutineScope {
-        val brevbaker = async { hentBrevakerMaler().asSequence().map { LetterMetadata.Brevbaker(it) } }
+        val brevbaker = async { hentBrevakerMaler().asSequence().filter { it.sakstyper.contains(sakstype) }.map { LetterMetadata.Brevbaker(it) } }
```

Comment:
Trengs det ein ekstrasjekk for _ALL_?

Reply:
all er et Set med alle enumerasjonene ja.